test/CollapsibleSpec.js

```diff
@@ -31,6 +31,35 @@ describe('<Collapsible />', () => {
     assert(wrapper.find('[data-collapsible="accordion"]').length, 'with a data attribute');
   });
 
+  it('should work with null dynamic children', () => {
+    let wrapper = mount(
+      <Collapsible accordion>
+        {null}
+        <CollapsibleItem header='First'>
+          Lorem ipsum dolor sit amet.
+        </CollapsibleItem>
+      </Collapsible>
+    );
+
+    assert.strictEqual(wrapper.find('li').length, 1);
+  });
+
+  it('should pass the key props to its children', () => {
```

Comment:
Not sure if this description is very meaningful reading the assertion, could you test if the child receives the key separately from testing that it doesn't render duplicates, since that's what it seems you're testing